launchable/commands/record/build.py

```diff
@@ -69,6 +69,8 @@ def build(ctx: click.core.Context, build_name: str, source: List[str], max_days:
 
     if "/" in build_name or "%2f" in build_name.lower():
         sys.exit("--name must not contain a slash and an encoded slash")
+    if "%25" in build_name:
+        sys.exit("--name must not contain %25")
```

Comment:
How about `"--name must not contain encoded % (%25)"`?

Reply:
fixed
https://github.com/launchableinc/cli/pull/530/commits/5364a285aefafed554a6a77ce36d60198bf16253